? config.sub
? libtool
? ltmain.sh
? ylwrap
? libffi/Makefile.in
? libffi/aclocal.m4
? libffi/configure
? libffi/fficonfig.h.in
? libffi/include/Makefile.in
? libgc/Makefile.in
? libgc/aclocal.m4
? libgc/config.guess
? libgc/config.sub
? libgc/configure
? libgc/ltmain.sh
? libgc/doc/Makefile.in
? libgc/include/Makefile.in
Index: configure.in
===================================================================
RCS file: /sources/dotgnu-pnet/pnet/configure.in,v
retrieving revision 1.229
diff -u -r1.229 configure.in
--- configure.in	9 Dec 2007 21:17:20 -0000	1.229
+++ configure.in	18 May 2008 11:43:34 -0000
@@ -262,7 +262,7 @@
 	*-*-darwin*)
 			THREADS=posix
 			;;
-	*-*-beos*)
+	*-*-beos*|*-*-haiku*)
 			THREADS=beos
 			;;	
 	esac
@@ -375,6 +375,7 @@
 fi
 AC_CHECK_LIB(dl, dlopen)
 AC_CHECK_LIB(socket, connect)
+AC_CHECK_LIB(network, connect)
 AC_CHECK_LIB(nsl, gethostbyname)
 AC_CHECK_LIB(rt, clock_gettime)
 have_libz=no
